manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -85,13 +105,14 @@ func (r *FybrikApplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	// check if webhooks are enabled and application has been validated before or if validated application is outdated
 	if os.Getenv("ENABLE_WEBHOOKS") != "true" && (string(applicationContext.Status.ValidApplication) == "" || observedStatus.ValidatedGeneration != appVersion) {
 		// do validation on applicationContext
+		log.Trace().Str(logging.FYBRIKAPPUUID, uuid).Msg("Reconciler validating Fybrik application")
 		err := applicationContext.ValidateFybrikApplication("/tmp/taxonomy/fybrik_application.json")
-		log.V(0).Info("Reconciler validating Fybrik application")
 		applicationContext.Status.ValidatedGeneration = appVersion
+
 		// if validation fails
 		if err != nil {
 			// set error message
-			log.V(0).Info("Fybrik application validation failed " + err.Error())
+			log.Fatal().Err(err).Str(logging.FYBRIKAPPUUID, uuid).Bool(logging.FORUSER, true).Bool(logging.AUDIT, true).Msg("FybrikApplication valdiation failed")
```

Comment:
Lack of consistency. If a webhook rejects the resource, no audit entry is created.

Reply:
@shlomitk1 We could add logging in the webhook